tools/public_api_guard/cdk/testing.d.ts

```diff
@@ -11,6 +11,11 @@ export interface BaseHarnessFilters {
     selector?: string;
 }
 
+export interface ChangeDetectionBatchingStatus {
+    isBatching: boolean;
```

Comment:
After writing the documentation for harness environment authors, I kind of wonder if we should rename this interface to `AutoChangeDetectionStatus` and this property to `isDisabled`. Thoughts?

Reply:
sgtm